config/experiments.go

```diff
@@ -1,12 +1,14 @@
 package config
 
 type Experiments struct {
-	FilestoreEnabled     bool
-	UrlstoreEnabled      bool
-	ShardingEnabled      bool `json:",omitempty"` // deprecated by autosharding: https://github.com/ipfs/kubo/pull/8527
-	GraphsyncEnabled     bool
-	Libp2pStreamMounting bool
-	P2pHttpProxy         bool //nolint
-	StrategicProviding   bool
-	AcceleratedDHTClient bool
+	FilestoreEnabled              bool
+	UrlstoreEnabled               bool
+	ShardingEnabled               bool `json:",omitempty"` // deprecated by autosharding: https://github.com/ipfs/kubo/pull/8527
+	GraphsyncEnabled              bool
+	Libp2pStreamMounting          bool
+	P2pHttpProxy                  bool //nolint
+	StrategicProviding            bool
+	AcceleratedDHTClient          bool
+	OptimisticProvide             bool
+	OptimisticProvideJobsPoolSize int
```

Comment:
Two questions:
1. Can you also add explanations of these into the docs folder?
2. Are these supposed to be global or per-DHT router? (or maybe they're supposed to be per-dht but that's pending the fixes to custom routing)

Reply:
1) Yes @dennis-tra is going to fill out the docs for this

2) I think all we _really_ care about is the WAN DHT here right? I've tried to just enable opt prov on everything when this is enabled, _unless_ you are using a `custom` router.

Reply:
Added the description 👍